Filters/exceptions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdguardFilters/issues/193777#issuecomment-2507572485
```

Comment:
Is `#issuecomment-2507572485` needed here, or perhaps it can be removed?

Reply:
Just pointed right to the video.

Reply:
I think it's ok to leave just the original link as you said earlier, so I've done it.